apps/labs/posts/labs-status-update-2019-04.md

```diff
@@ -0,0 +1,188 @@
+do the [numpydoc](https://github.com/numpy/numpydoc) 0.9 release, deal
+with several build issues, and review a number of PRs
+(the one
+allowing to specify [BLAS and LAPACK link order](https://github.com/numpy/numpy/pull/13132)
+was particularly nice). On the organizational front, I fixed the description
+of how donations are handled on numpy.org, finalized the
+[Tidelift](https://tidelift.com/) agreement for NumPy (see the
+[announcement](https://mail.python.org/pipermail/numpy-discussion/2019-April/079370.html)
+for details), helped NumPy and SciPy get accepted for the
+[Google Season of Docs](https://developers.google.com/season-of-docs/) program,
+and did everything needed to finalize the fiscal sponsorship agreement between
+SciPy and NumFOCUS.
+
+## Jupyter and JupyterLab improvements
+
+Jupyter is a key part of the PyData ecosystem. It extends well beyond that though, so I'm
+giving it its own bucket here. At Quansight we have a number of Jupyter core developers
+and contributors. Ian Rose, Saul Shanabrook, Grant Nestor and others have been very busy
+with both maintenance tasks and adding new features to Jupyter and JupyterLab.
+
+JupyterLab is about to get support for printing (not inside the notebook, but the old-fashioned
+`Ctrl-P` variant). [This pull request](https://github.com/jupyterlab/jupyterlab/pull/5850)
+by Saul has nice screenshots showing the feature in action for whole notebooks,
+ images, the JSON viewer and the inspector.
+
+Ian worked on the third alpha release of JupyterLab 1.0, on testing and CI infrastructure,
+and other general maintenance tasks. He also improved PDF preview in JupyterLab, so it
+now [works as expected](https://github.com/jupyterlab/jupyterlab/pull/6264) in Firefox
+and Chrome (at least).
+
+Saul added support for the [nteract Data Explorer](https://github.com/nteract/nteract/tree/master/packages/data-explorer) to the JupyterLab data registry as a plugin.
+[This pull request](https://github.com/jupyterlab/jupyterlab-data-explorer/pull/10) shows it
+in action on a pandas DataFrame.
+
+Other interesting features are in progress and will make their way into the main
+repositories soon.
+
+## Starting to shape Labs
+
+There is a lot of work to do to figure out for ourselves exactly what Labs
+will be, and then to communicate that clearly to the outside world. We have
+a rough idea (see [my first blog post](https://labs.quansight.org/blog/2019/04/joining-labs/)
+and [Travis' blog post](https://www.quansight.com/single-post/2019/04/02/Welcoming-Ralf-Gommers-as-Director-of-Quansight-Labs)), but there's a long way
+to go from there to having an compelling elevator pitch, a website that tells
+our story well, people and projects organized, a funding stream, and more.
+
+One of the first things we did do is start this blog, to start communicating
+about the technical work we're doing. We're also going through the roadmaps
+published at [quansight.com/projects](https://www.quansight.com/projects),
```

Comment:
```suggestion
```

Reply:
@rgommers I need to remove this link. Is this edit acceptable?

Reply:
yes that's fine